cypress/integration/smokeTest.js

```diff
@@ -1,32 +1,99 @@
 import {
   formatCode,
   typeCode,
-  assertFrameContains,
+  getFirstFrame,
+  assertFirstFrameContains,
   assertCodePaneContains,
-  assertCodePaneLineCount
+  assertCodePaneLineCount,
+  assertFramesMatch,
+  selectWidthPreferenceByIndex
 } from '../support/utils';
 
 describe('Smoke test', () => {
-  it('works', () => {
+  it('renders to frame', () => {
     typeCode('<Foo />');
-    assertFrameContains('Foo');
+    assertFirstFrameContains('Foo');
 
     typeCode('<Bar />');
-    assertFrameContains('Bar');
+    assertFirstFrameContains('Bar');
   });
 
   it('autocompletes', () => {
     typeCode('<F{enter} c{enter}={downarrow}{enter} />');
-    assertFrameContains('Foo');
+    assertFirstFrameContains('Foo');
     assertCodePaneContains('<Foo color="blue" />');
   });
 
   it('formats', () => {
-    cy.visit(
-      'http://localhost:9000/#?code=PEZvbz48Rm9vPjxCYXIvPjwvRm9vPjwvRm9vPg'
-    ).reload();
+    cy.visit('http://localhost:9000/').reload();
+
+    typeCode('<Foo><Foo><Bar/>');
     assertCodePaneLineCount(1);
     formatCode();
     assertCodePaneLineCount(6);
   });
+
+  it('url handling - code (base64 encoding)', () => {
```

Comment:
So glad this is covered now! 👏

Reply:
ironically, this was covered before, but the new encoding wasnt 😄

Reply:
lol 😅